docker-compose.yml

```diff
@@ -51,6 +51,34 @@ services:
   #     - '3306:3306'
   #   volumes:
   #     - hapi-fhir-mysql:/var/lib/mysql
+  hapi-fhir-keycloak:
+    image: quay.io/keycloak/keycloak:22.0.5
+    container_name: keycloak
+    restart: on-failure
+    ports:
+      - '8085:8085'
+    command:
+      - "start-dev"
+    environment:
+      KC_ADMIN: 'admin'
+      KC_ADMIN_PASSWORD: 'admin'
+      KC_DB: 'postgres'
+      KC_DB_URL_HOST: 'hapi-fhir-postgres'
```

Comment:
I don't think that the two different containers can use the same postgres.

Reply:
I will add another postgress container for keycloak